lib/mapper.js

```diff
@@ -32,6 +34,9 @@ exports.identify = function(msg){
   if (msg.email()) ret.email = msg.email();
   if (msg.name()) ret.name = msg.name();
 
+  // Add location data
+  if (msg.context().location) ret.location_data = formatLocation(msg.context());
```

Comment:
Rather than calling `context()` multiple times you should store the result of calling it in a variable